### DISCUSSION TITLE
add code snippet in textarea only works after key stroke

### DISCUSSION THREAD

**Discussion by hermanvl** [PrismJS/prism#2916] [Q&A]
Hello, I am using prism live in a textarea and it works fine but when I try to add a code snippet using a button onclick function (document.getElementById("content").innerHTML += header;), the code is added but invisible until i press spacebar or another key, then it shows up nicely highlighted as expected. Is there a way to force it so it is highlighted right away?
It works when I just paste it with ctrl-v but not with button onclick.
thanks

**Comment by RunDevelopment** [comment]
I'm sorry but this is not the right place to ask this question. You correctly opened PrismJS/live#17. A maintainer will look at the issue over there.
